packages/sitecore-jss-react/src/components/ErrorBoundary.test.tsx

```diff
@@ -243,6 +243,50 @@ describe('ErrorBoundary', () => {
     });
   });
   describe('when not in page editing and not in development mode', () => {
+    const delay = (timeout, promise?) => {
+      return new Promise((resolve) => {
+        setTimeout(resolve, timeout);
+      }).then(() => promise);
+    };
+
+    const ItsADynamicComponent = React.lazy(() =>
+      delay(500, import('../test-data/test-dynamic-component'))
+    );
+
+    it('should render a loading message', async () => {
+      const rendered = mount(
+        <ErrorBoundary>
+          <ItsADynamicComponent />
+        </ErrorBoundary>
+      );
+      expect(rendered.text()).to.equal('Loading component...');
+    });
+
+    it('should render custom loading message', async () => {
+      const loading = 'I am customly loading...';
+      const rendered = mount(
+        <ErrorBoundary componentLoadingMessage={loading}>
+          <ItsADynamicComponent />
+        </ErrorBoundary>
+      );
+      expect(rendered.text()).to.equal(loading);
+    });
+
+    it('should not render Suspense and default loading message when wrapping a dynamic component', async () => {
+      // mount fails with lazy component and no suspense
+      const rendered = mount(
+        <Suspense>
```

Comment:
Does this mean we have to wrap all the dynamic components with <Suspense>?

Reply:
for tests, yes
mount here refers to enzyme mount() specifically